### ISSUE TITLE
man facepalming emoji brings woman facepalming

### ISSUE THREAD

**Issue by THausherr** [johannhof/emoji-helper#132]
How to reproduce:

enter facepalm in search
![grafik](https://user-images.githubusercontent.com/6665575/78469770-5a8dec80-7724-11ea-962b-15a4cfd0cd61.png)

click on the man
![grafik](https://user-images.githubusercontent.com/6665575/78469782-73969d80-7724-11ea-9cf2-60d812a433a0.png)

click on the image on the right, then paste in twitter, see a woman facepalming glyph and a "man icon"
![grafik](https://user-images.githubusercontent.com/6665575/78469817-b6f10c00-7724-11ea-9661-717114de2bb0.png)

What I expected is the glyph from here
https://emojipedia.org/man-facepalming/

which is 🤦‍♂️

I am using version 2.0.1 on firefox 74.0.1.

**Comment by LeoniePhiline** [comment]
This extension appears to be unmaintained. Try https://github.com/johannhof/emoji-helper/issues/127#issuecomment-498349096

**Comment by THausherr** [comment]
Yes, after creating the issue I realized that it is probably futile. But the other extension brings a warning that it has not been evaluated by Mozilla.
